install/sensors.csv

```diff
@@ -816,6 +816,8 @@ Lennartz Electronic,LE-3DliteMkIII,J-0947,AMCZ,10,0,,0,0,0,0,0,0,2024-09-16T05:0
 Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06812,NA002,04,0,,0,0,0,0,0,0,2024-03-21T02:10:00Z,9999-01-01T00:00:00Z
 Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06919,TO006,05,0,,0,0,0,0,0,0,2024-06-04T03:59:59Z,9999-01-01T00:00:00Z
 Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06920,TO003,12,0,,0,0,0,0,0,0,2024-03-01T00:20:00Z,9999-01-01T00:00:00Z
+Nanometrics,Cascadia Titan TACCL-PH1,3515,WLCZ,21,0,,0,15.5,0,0,0,0,2024-10-16T03:30:00Z,9999-01-01T00:00:00Z
+Nanometrics,Cascadia Trillium TC120PH-1,3515,WLCZ,10,0,,0,15.5,0,0,0,0,2024-10-16T03:30:00Z,9999-01-01T00:00:00Z
```

Comment:
please double check the serial number, thanks.

Reply:
@mnaguit  the serial number(s) is actually correct. The strong motion and weak motion is housed together in a single casing. 

@Thomas-Benson  I made a mistake in the Integration testing ticket.  The down-hole Titan is called TACCl-BH1 so the this should probably called **Cascadia Titan TACCL-BH1**  ( I did fix in the ticket also....)

Reply:
Thanks @gva6500 , had to double check coz the ticket says the serial number is `3512` instead of `3515`. just a typo in the ticket?

Reply:
You are 100% correct! @mnaguit  
 my short stubby fingers make lots of typos 🙃